app/models/category.rb

```diff
@@ -0,0 +1,3 @@
+class Category < ApplicationRecord
+    has_one :video
```

Comment:
How category can have one video?

Reply:
category have many videos & videos belongs to category. issue resolved